runelite-client/src/main/java/net/runelite/client/plugins/friendnotes/FriendNotesPlugin.java

```diff
@@ -84,19 +99,74 @@
 	@Inject
 	private ChatboxPanelManager chatboxPanelManager;
 
+	@Inject
+	private ClientThread clientThread;
+
+	@Inject
+	private FriendNotesConfig config;
+
 	@Getter
 	private HoveredFriend hoveredFriend = null;
 
+	private int iconIdx = -1;
+	private String currentlyLayouting = null;
```

Comment:
This is the default initialized value of this field, it does not need to be explicitly defined.
```suggestion
	private String currentlyLayouting;
```

Reply:
changed